### PR TITLE
Brigcells fix port

### PR DESCRIPTION
**Pull request by Uqualm**
There we go.  It should work.

### CODE REVIEW

File: code/game/machinery/doors/brigdoors.dm [hunk 1]
```diff
@@ -155,11 +155,16 @@
 
 /obj/machinery/door_timer/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = 0, \
 										datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
-	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
-	if(!ui)
-		ui = new(user, src, ui_key, "brig_timer", name, 300, 200, master_ui, state)
-		ui.open()
 
+	if(!allowed(usr))
+		usr << "<span class='danger'>Access denied.</span>"
```
Comment (tortellinitony) [inline code/game/machinery/doors/brigdoors.dm:160]: Use to_chat

**Comment by tortellinitony** [comment]
It seems there is no need for this PR, as you cannot do anything with the screens without access.

**Comment by Uqualm** [comment]
Yes I thought it was stupid to, but it's on the to do list so I did it. Remove it from the to do list and I'll close it.

**Comment by tortellinitony** [comment]
Alright then, since /tg/ already took care of this I'll check it off.

**Pull request closed without merging**